ouroboros-consensus/ouroboros-consensus.cabal

```
@@ -324,7 +326,13 @@ library
 
 
   if os(windows)
-     build-depends:    Win32 >= 2.6.2.0
+    if impl(ghc >= 8.10.2) && impl(ghc < 8.12)
```

```suggestion
    if impl(ghc >= 8.10.2) && impl(ghc < 8.11)
```
There is no `ghc-8.11` isn't it?

---

I've moved it to `< ghc-8.11`, although I've never seen a `ghc-8.11` in the wild. 😁